spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -132,6 +132,17 @@ def associate_groups_with_app(app, groups)
       details_for_app(app)
     end
 
+    def valid_name_for(input)
+      latinazed = input.to_slug.transliterate.to_s # remove accents
+      latinazed = latinazed.gsub(/[^0-9A-Za-z\d\s]/, '') # remove non-valid characters
+      # it may result in totaly empty strings that only contain chinese symbols.
+      if latinazed != input
```

Comment:
The only reason the md5 would need to be added would be if the latinized version of the app name is empty, right? If so, let's change the condition to `latinazed.strip.empty?`